strawberryfields/backends/gaussianbackend/gaussiancircuit.py

```diff
@@ -77,7 +77,7 @@ def add_mode(self, n=1):
         self.nlen = newnlen
 
     def del_mode(self, modes):
-        """ delete mode from the circuit"""
+        """delete mode from the circuit"""
```

Comment:
Black now formats docstrings??? 🙏 
This makes me so happy. It always bothered me that it ignored them